bag.map

```diff
@@ -250,10 +250,349 @@ MAP
+      "wms_include_items"   "all"                                        # Toon alle attributen in het CSV antwoord (WMS)
+    END
+
+    CLASS
+      NAME                  "bouw_gestart"           # Lowercase naam in enkelvoud zonder afkortingen, zonder spaties en zonder leestekens (apostrofe)
+      TITLE                 "Bouw gestart"    # Sentence case naam in enkelvoud met eventuele afkorting in haakjes erachter
+      STYLE
+        ANTIALIAS           true
+        COLOR               "#8B5A00"
+        OPACITY             20
+      END
+      STYLE
+        OUTLINECOLOR        "#8B5A00"
+        WIDTH               2
+      END
+    END
+
+  END
+
+
+
+  LAYER
+    NAME                    "panden_naar_status_bouwvergunning_verleend"         # Lowercase naam in meervoud zonder afkortingen, zonder spaties en zonder leestekens (apostrofe)
+    INCLUDE                 "connection/dataservices.inc"
+    GROUP                   "pand_naar_status"                                 # Minstens 1 zelfde naam als de mapfile, of meerdere (slugified)
+    DATA                    "geometrie FROM (
+            SELECT id, geometrie,status_omschrijving FROM
+                public.bag_panden
+            WHERE
+                begin_geldigheid < now()
+                AND (eind_geldigheid IS NULL OR eind_geldigheid > now())
+                ) AS sub
+        USING srid=28992 USING UNIQUE id"
+
+    FILTER                  ([status_omschrijving] = 'Bouwvergunning verleend')            # Dit wellicht in WHERE clause, uitzoeken
+    TYPE                    POLYGON
+    TEMPLATE                "empty"
+    PROJECTION
+      "init=epsg:28992"
+    END
+
+    METADATA
+      "ows_title"           "Panden naar Status: Bouwvergunning Verleend" # Sentence case naam in meervoud met eventuele afkorting in haakjes erachter
+      "ows_abstract"        "Actieve BAG panden naar Status Bouwvergunning Verleend" # Door datateam aanleveren
+      "wms_group_title"     "Panden naar Status"                                 # Sentence case naam in meervoud met eventuele afkorting in haakjes erachter
+      "gml_featureid"       "id"                                         # Unique ID volgens het Amsterdam Schema
+      "gml_geometries"      "geometry"
+      "gml_geometry_type"   "multipolygon"                                      # Gelijk aan de TYPE van de LAYER
+      "gml_include_items"   "all"                                        # Toon alle attributen in het GML antwoord (WMS én WFS)
+      "gml_types"           "auto"
+      "wms_include_items"   "all"                                        # Toon alle attributen in het CSV antwoord (WMS)
+    END
+
+    CLASS
+      NAME                  "pand_bouwvergunning_verleend"           # Lowercase naam in enkelvoud zonder afkortingen, zonder spaties en zonder leestekens (apostrofe)
+      TITLE                 "Pand Bouwvergunning verleend"    # Sentence case naam in enkelvoud met eventuele afkorting in haakjes erachter
```

Comment:
De waarde van het attribuut is 'Bouwvergunning verleend'. Dus de CLASS NAME en TITLE moeten zonder 'pand' ervoor.